### DISCUSSION TITLE
error in simulation

### DISCUSSION THREAD

**Discussion by YazanBaddour** [lnis-uofu/OpenFPGA#1518] [General]
hello!!

i am trying to implement this rtl 
![image](https://github.com/lnis-uofu/OpenFPGA/assets/153017100/d43a782f-d7a2-4a40-9c6c-3060dc07de45)
when simulating the fabric generated by OpenFPGA i get the following : 
![image](https://github.com/lnis-uofu/OpenFPGA/assets/153017100/8ac83079-7c7f-4c99-948d-2cb300f2f57c)
the output doesn't change on negative edge reset(like the ref )  and the value is updated only on the following posedge of the clk 
note i  added constant_net_method route for earlier mismatches but i still have this issue that output doesn't change on negative edge 
i am using the write_full_testbench_example_script.openfpga for the script  and for the architecture 
[xmlarch.txt](https://github.com/lnis-uofu/OpenFPGA/files/13942399/xmlarch.txt)
[openfpgaarch.txt](https://github.com/lnis-uofu/OpenFPGA/files/13942400/openfpgaarch.txt)

 
any help would be appreciated how can i make my fabric outputs sensitive to negative edge signals ?
thank you

**Comment by tangxifan** [comment]
@YazanBaddour 

Please refer to the example: https://github.com/lnis-uofu/OpenFPGA/tree/master/openfpga_flow/tasks/basic_tests/k4_series/k4n4_fracff2edge/config

**Reply by YazanBaddour** [reply]
thank you! 
i tried running another design into the openfpga flow which is a finite state machine and the openfpga gives me error (only rising edges latches are supported ) 
![image](https://github.com/lnis-uofu/OpenFPGA/assets/153017100/4a591e36-c838-4e25-a85e-7f590f375a75)
is there anyway to fix this error 
thank you
